mmdet/models/necks/bifpn.py

```diff
@@ -0,0 +1,326 @@
+# Based off of github.com/rwightman/efficientdet-pytorch
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import xavier_init
+from mmdet.core import auto_fp16
+
+from ..builder import NECKS
+
+from mmcv.cnn import normal_init
+
+act_fn_list = ["silu", "swish", "hswish", "relu", "relu6", "mish", "srelu"]
+
+class ActLayer(nn.Module):
+    def __init__(self, act_name):
+        super(ActLayer, self).__init__()
+        assert act_name in act_fn_list
+        self.act_fn = act_name
+
+    def forward(self, nodes):
+        # Activation function
+        if (self.act_fn == "silu"):
+            nodes = nodes * torch.sigmoid(nodes)
+
+        # # Quantization-friendly hard swish
```

Comment:
Do you use ALL following activations somewhere? Why do you bring them here? You should consider registering new activation functions using mmcv activations registry in order to use them with default mmcv cnn building blocks. https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/bricks/activation.py (this is for example, pay attention to version that you are using)

Reply:
All activation functions are provided in the [original implementation]https://github.com/google/automl/blob/262754b807610f947d57e0772757a8985b3b4895/efficientdet/utils.py#L37l). I will consider making a PR for the activation functions, but would prefer to not have this PR be dependent on another one.